cli/src/main/java/io/kestra/cli/commands/plugins/PluginSearchCommand.java

```diff
@@ -0,0 +1,173 @@
+package io.kestra.cli.commands.plugins;
+
+import io.kestra.cli.AbstractCommand;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+@Command(
+    name = "search",
+    description = "Search for available Kestra plugins"
+)
+public class PluginSearchCommand extends AbstractCommand {
+    private static final String API_URL = "https://api.kestra.io/v1/plugins";
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+    private static final HttpClient CLIENT = HttpClient.newBuilder()
+        .connectTimeout(Duration.ofSeconds(10))
+        .build();
+    private static final char SPACE = ' ';
+
+    @Parameters(index = "0", description = "Search term (optional)", defaultValue = "")
+    private String searchTerm;
+
+    @Override
+    public Integer call() throws Exception {
+        super.call();
+
+        try {
+            JsonNode root = fetchPlugins();
+            if (root == null) return 1;
+
+            List<PluginInfo> plugins = findPlugins(root);
+            printResults(plugins);
+            return 0;
+
+        } catch (Exception e) {
+            stdOut("Error processing plugins: {0}", e.getMessage());
+            return 1;
+        }
+    }
+
+    private JsonNode fetchPlugins() throws Exception {
+        HttpRequest request = HttpRequest.newBuilder()
+            .uri(URI.create(API_URL))
+            .header("Accept", "application/json")
+            .GET()
+            .build();
+
+        HttpResponse<String> response = CLIENT.send(request, HttpResponse.BodyHandlers.ofString());
+
+        if (response.statusCode() != 200) {
+            stdOut("API request failed with status: {0}", response.statusCode());
+            return null;
+        }
+
+        return MAPPER.readTree(response.body());
+    }
+
+    private List<PluginInfo> findPlugins(JsonNode root) {
+        String searchTermLower = searchTerm.toLowerCase();
+        List<PluginInfo> plugins = new ArrayList<>();
+
+        for (JsonNode plugin : root) {
+            if (matchesSearch(plugin, searchTermLower)) {
+                plugins.add(new PluginInfo(
+                    plugin.path("name").asText(),
+                    plugin.path("title").asText(),
+                    plugin.path("group").asText(),
+                    plugin.path("version").asText("")
+                ));
+            }
+        }
+
+        plugins.sort((p1, p2) -> p1.name.compareToIgnoreCase(p2.name));
+        return plugins;
+    }
+
+    private boolean matchesSearch(JsonNode plugin, String term) {
+        if (term.isEmpty()) return true;
```

Comment:
Please always use `{` and `}` for if statements to facilitate future refactoring